shared/src/iosMain/kotlin/co/touchlab/kampkit/BreedCallbackViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package co.touchlab.kampkit
+
+import co.touchlab.kampkit.db.Breed
+import co.touchlab.kampkit.models.BreedRepository
+import co.touchlab.kampkit.models.BreedViewModel
+import co.touchlab.kampkit.models.CallbackViewModel
+import co.touchlab.kermit.Logger
+
+@Suppress("Unused") // Called from Swift
```

Comment:
Yeah that's a good point. This suppression can probably be dropped.

Reply:
Actually, the reason I did this was so the member functions wouldn't get marked unused, since they are only called from Swift. Updated the comment to make this more explicit